system/include/webgpu/webgpu.h

```diff
@@ -860,10 +958,6 @@ typedef void (*WGPUProc)(void);
 typedef WGPUInstance (*WGPUProcCreateInstance)(WGPUInstanceDescriptor const * descriptor);
 typedef WGPUProc (*WGPUProcGetProcAddress)(WGPUDevice device, char const * procName);
 
-// Procs of Adapter
-typedef void (*WGPUProcAdapterGetProperties)(WGPUAdapter adapter, WGPUAdapterProperties * properties);
-typedef void (*WGPUProcAdapterRequestDevice)(WGPUAdapter adapter, WGPUDeviceDescriptor const * descriptor, WGPURequestDeviceCallback callback, void * userdata);
-
```

Comment:
Should keep these

Reply:
Ok, I put those back, although just to be sure, none of those are implemented in `library_webgpu.js`